### PR TITLE
feat(Unleash::Incognia): add public Client

### PR DESCRIPTION
**Pull request by julianalucena**
Adds the possibility to list enabled features and verify if some
feature is enabled.

### CODE REVIEW

File: lib/unleash/incognia/client.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      def initialize(user:, organization: nil)
+        @user = user
+        @organization = organization
+      end
+
+      def feature_enabled?(name)
+        client.is_enabled? name, context
+      end
+
+      def enabled_features
```
Comment (ottony) [inline lib/unleash/incognia/client.rb:13]: Nice

File: README.md [hunk 2]
```diff
@@ -24,6 +24,43 @@ Or install it yourself as:
...
+
+### Configuration
+
+Whenever you have `Unleash::Client` being instanciated change it to:
+
+```ruby
+Unleash::Incognia.configure do |config|
+  config.unleash_client = Unleash::Client.new
+end
+```
```
Comment (ottony) [inline README.md:60]: If you want to mention here, there's a second way to configure too

```ruby
Unleash::Incognia.configuration.unleash_client = Unleash::Client.new
```

**Review by ottony — APPROVED** [review]
Simple and direct 👏

File: README.md (hunk 2 above)
Comment (julianalucena) [reply README.md:60]: Done!

**Pull request merged**